src/debt_order/index.ts

```diff
@@ -175,6 +194,10 @@ export class DebtOrder {
         );
     }
 
+    private getAgreementId(): string {
```

Comment:
I think we decided to go with `issuanceHash` across the dharma.js codebase?  And if we haven't decided one way or another, I think we should pick one of `agreementId` and `issuanceHash`

Reply:
😅